### PR TITLE
Feature bl 6016 unsubscribe network charges

### PR DESCRIPTION
**Pull request by ibrahimjassat**
added in unsubscribe network charges disclaimer content for SMS

### CODE REVIEW

**Comment by ibrahimjassat** [comment]
@Jerotire Spoke to Joel and the <strong> is perfect and I will squash the commits

File: webapp/src/main/resources/template/terms-and-conditions.hbs [hunk 1]
```diff
@@ -40,6 +40,10 @@
         <p>Where we identify that an email address provided does not exist
             anymore, we will automatically delete the subscription.</p>
 
+        <p><strong>Unsubscribing from MOT reminders</strong></p>
```
Comment (raffcecco) [inline webapp/src/main/resources/template/terms-and-conditions.hbs]: Not sure if `<p><strong>` should be used for a heading. Semantically this is wrong as it defines a paragraph, not a heading. I was under the impression this should be an in a new section, therefore an `<h2>`. I don't see the pattern of `<p><strong>` used for a heading in any of the existing templates. I suspect there would be accessibility/GDS issues with this.

Comment (joelspuds) [reply webapp/src/main/resources/template/terms-and-conditions.hbs]: <h3 class="heading-small">Unsubscribing from MOT reminders</h3>

Comment (mathieuhendey) [reply webapp/src/main/resources/template/terms-and-conditions.hbs]: Seems like every other heading on the page uses `heading-medium` @joelspuds

Comment (raffcecco) [reply webapp/src/main/resources/template/terms-and-conditions.hbs]: Check the ticket https://jira.i-env.net/browse/BL-6016. There are now screen grabs which show the two screens. The HTML now matches those.

**Pull request merged**